feature-examples/python/api_key/list_api_key.py

```diff
@@ -0,0 +1,90 @@
+"""Example of using the Vectara REST API to list the API Keys."""
+
+import dataclasses
+import logging
+
+import requests
+
+
+@dataclasses.dataclass(frozen=True)
+class CorpusData:
+    """Corpus id and name."""
+
+    corpus_id: int
+    corpus_name: str
+
+
+@dataclasses.dataclass(frozen=True)
+class KeyData:
+    """API Key data such as id, type, list of corpora etc."""
+
+    key_id: str
+    description: str
+    key_type: str
+    enabled: bool
+    corpora: list[CorpusData]
+
+
+def list_apikeys(
+    customer_id: int,
+    jwt_token: str,
+) -> list[KeyData]:
+    """Retrieves the list of API keys.
+
+    Args:
+        customer_id: Unique customer ID in vectara platform.
+        jwt_token: JWT token to be used for authentication.
+
+    Returns:
+        (message, True) in case of success and returns (status, False) in case of failure.
```

Comment:
Please update the return type in the doc and add a `Raises:` section.
https://google.github.io/styleguide/pyguide.html#doc-function-raises